reducers/views/channel_sidebar.ts

```diff
@@ -75,8 +75,62 @@ export function newCategoryIds(state: string[] = [], action: GenericAction): str
     }
 }
 
+export function selectedChannelIds(state: string[] = [], action: GenericAction): string[] {
+    switch (action.type) {
+    case ActionTypes.MULTISELECT_CHANNEL:
+        // Task was not previously selected
```

Comment:
Can you update these comments to refer to channels? It's a bit confusing to see task-related stuff that I assume that came from some demo code